packages/plugin-ext/src/plugin/tasks/task-provider.ts

```diff
@@ -50,9 +44,7 @@ export class TaskProviderAdapter {
         if (typeof this.provider.resolveTask !== 'function') {
             return Promise.resolve(undefined);
         }
-        const id = ObjectIdentifier.of(task);
-        const cached = this.cache.get(id);
-        const item = cached ? cached : Converter.toTask(task);
+        const item = Converter.toTask(task);
```

Comment:
Could you please explain why we need to remove the caching mechanism ?

Reply:
We need to remove this cache system to allow executing of temporary modified tasks.
Otherwise it will ignore changes and use not the task which comes, but cached one.